docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -296,6 +297,52 @@ To enable denormals optimization in the application, the `denormals_optimization
 
 @endsphinxdirective
 
+### Sparse weights decompression
+`Sparse weights` are weights where most of the elements are zero. The ratio of the number of zero elements to the number of all elements is called `sparse rate`. Thus, we assume that `sparse weights` are weights with a high sparse rate. In case of `sparse weights`, we can store only non-zero values in memory using special storage structures, which allows us to use memory more efficiently. In turn, this can give us better performance in the high memory bound workloads (e.g., throughput scenario).
+
+`Sparse weights decompression feature` allows to pack weights for Matrix Multiplication operations directly in the CPU plugin at the model compilation stage and store non-zero values in a special packed format. Then, during the execution of the model, the weights are unpacked and used in the computational kernel.
+
+To use this feature, the user is provided with property `sparse_weights_decompression_rate`, which can take values from the interval [0, 1]. `sparse_weights_decompression_rate` is the minimum fraction of nulls above which we use weights compression. The default value is `1`, which means the option is disabled.
```

Comment:
"is the minimum fraction of nulls above which we use weights compression" -> "defines sparse rate threashold: only operations with higher sparse rate will be executed using sparse weights decompression feature"

Reply:
fixed, thanks